test/e2e/suite/command/resolve.go

```diff
@@ -40,14 +49,20 @@ var _ = Describe("ORAS beginners:", func() {
 		It("should fail when provided manifest reference is not found", func() {
 			ORAS("resolve", RegistryRef(ZOTHost, ImageRepo, "i-dont-think-this-tag-exists")).ExpectFailure().MatchErrKeyWords("Error: failed to resolve digest:", "not found").Exec()
 		})
+
+	})
+})
+
+var _ = Describe("Common registry user", func() {
+	When("running resolve command", func() {
 		It("should resolve with just digest", func() {
 			out := ORAS("resolve", RegistryRef(ZOTHost, ImageRepo, multi_arch.Digest)).Exec().Out
 			outString := string(out.Contents())
 			outString = strings.TrimSpace(outString)
 			gomega.Expect(outString).To(gomega.Equal(multi_arch.Digest))
 		})
 		It("should resolve with a fully qualified reference", func() {
-			out := ORAS("digest", "-l", RegistryRef(ZOTHost, ImageRepo, multi_arch.Tag)).Exec().Out
+			out := ORAS("resolve", "-l", RegistryRef(ZOTHost, ImageRepo, multi_arch.Tag)).Exec().Out
```

Comment:
I left one command to actually use the digest alias to make sure the command does execute rather than just the help.

Reply:
Covered alias case in `oras digest -h`

Reply:
If the help validates the output exactly to match the resolve help text then that’s ok. Otherwise it’s better to actually test the alias itself.

Reply:
Yes you are right, the experimental spec doesn't test anything specific to `oras resolve`. Reverting this change.